packages/api/Dockerfile

```diff
@@ -10,15 +10,16 @@ COPY infrastructure/package.json /app/infrastructure/package.json
 RUN yarn
 
 # ---- Development ----
+# Don't copy anything -- compose will do that.
 FROM base as development
 WORKDIR /app
-COPY . .
-RUN yarn
-RUN /app/node_modules/.bin/tsc
 
 # ---- Release ----
 FROM mhart/alpine-node:9.4.0 AS release
 WORKDIR /app
-COPY --from=development /app /app
+COPY . .
+COPY --from=base /app /app
+RUN yarn
+RUN /app/node_modules/.bin/tsc
```

Comment:
You'll need to adjust this pipeline.  

Create a release-build stage that does those 4 steps then have the release image just copy the dist into here.

Reply:
that will keep the release image tiny with only the compiled output

Reply:
:+1: